environments/acceptance/invoke.yml

```diff
@@ -0,0 +1,49 @@
+# NB: This file is committed in a public repo. DO NOT COMMIT REAL SECRETS!
+# It uses AWS Secret Manager to lookup any secrets at application bootstrap
+
+django:
+  debug: true
+  postgres_user: "postgres"  # TODO: change to "django" or other application user with low privileges
+  postgres_host: "surfpol-edushare.cikhmbah79es.eu-central-1.rds.amazonaws.com"
+  protocol: "http"
```

Comment:
Oepsie, wanneer kwam dat certificaat ook al weer? Wil liever niet weer lange tijd al mijn wachtwoorden publiekelijk

Reply:
Het gaat al over https. Alleen tot de loadbalancer. Binnen AWS is http